### ISSUE TITLE
OFF should show some random uncomplete product on "Edit Successful page"

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#4]
OFF should show some random uncomplete product on "Edit Successful page" to foster more contributions.

**Comment by hangy** [comment]
This is not the fastest query, but it gets a _random_ incomplete product:
```
> db.products.aggregate([{ $match: { "states_tags":"en:to-be-completed" } }, { $sample: { size: 1 } }])`
```

But I don't necessarily agree that I'd want to see a random incomplete product after saving a product. Maybe it should just show a random list of incomplete products, but also a link back to the product you were originally editing.

**Comment by teolemon** [comment]
Yes, that would more the idea. Perhaps a list of 10 similar products (10 products without brand) ? 
We have to think a little more about what will be likely to trigger another edit.

**Comment by hangy** [comment]
Working on this on https://github.com/openfoodfacts/openfoodfacts-server/tree/issue/4-product-after-edit

**Comment by hangy** [comment]
Maybe add some nice statement below "Changes saved"?
![WIP](https://cloud.githubusercontent.com/assets/87124/22386131/f4faa2a2-e4d5-11e6-9e1f-8c39831ab4da.PNG)

**Comment by offvince** [comment]
I for one would prefer to see the product I just edited (not a link to it but the product page itself).

The list of related products OTOH is a very powerful feature that might be displayed at all time on every product.

My 2c...

**Comment by teolemon** [comment]
@offvince What about putting the uncomplete products below the page of the product you just edited ?

**Comment by stephanegigandet** [comment]
We need to add some kind of message, otherwise it's hard to understand why there are products on top of the product page.

![image](https://user-images.githubusercontent.com/8158668/31026007-04eb285a-a545-11e7-8320-8e208732ab07.png)

**Comment by stephanegigandet** [comment]
This feature has been live on production for a few days, but it makes edit validation very slow (at least 10 seconds), I'm adding an option to disable for now.

**Comment by stephanegigandet** [comment]
Closing this issue as it's unclear what should be done and whether it should be done.
